plugin/gradle.properties

```diff
@@ -1,6 +1,6 @@
 # Maven configuration
 maven.groupId=com.here.gradle.plugins
 maven.artifactId=gradle-jenkins-jobdsl-plugin
-maven.version=3.1.0-SNAPSHOT
```

Comment:
Doesn't this also need to be changed from `3.1.0-SNAPSHOT` to `3.1.0` in the previous commit?

Reply:
No, actually the `-SNAPSHOT` is stripped from the value configured here as part of the release process.